store.ts

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit'
```

Comment:
I'd say that it'd better if we put store inside store folder instead of just one file since we will probably be adding RTK slices which will end up inside this folder instead of just at root directory